### DISCUSSION TITLE
Params Share/Modify on the fly

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#5631] [Q&A] [answered]
I would like to know if there is any suggested way to create or modify params on the fly when multiple runnables are running in parallel.
The goal of this is share information between multiple runnables at run time.

**Comment by richtja** [comment, marked as answer]
Hi @marioaag, avocado doesn't support changes to params during the runtime. Also, avocado runs each test in a separated process to isolate the test as much as possible, because of that avocado doesn't support any communication between tests. 

I am not really sure what your use-case is, but if you are using process spawner, you could use environment variables, for sharing information between tests. You can use [avocado variables](https://avocado-framework.readthedocs.io/en/latest/guides/writer/chapters/writing.html?highlight=enviroment%20variables#environment-variables-for-tests) or you can create your own in setup phase.

**Comment by marioaag** [comment]
Thanks for the confirmation!
